src/psyclone/psyir/backend/sympy_writer.py

```diff
@@ -420,7 +421,8 @@ def _create_type_map(self, list_of_expressions: Iterable[Node],
                 # have a scope, hence the try...except.
                 orig_sym = sva[0].node.scope.symbol_table.lookup(sig.var_name)
             except SymbolError:
-                if isinstance(sva[0].node, Reference):
+                if (isinstance(sva[0].node, Reference) and
+                        sva[0].access_type != AccessType.TYPE_INFO):
```

Comment:
Could you add a comment explaining why this is needed?

Reply:
Ok, so I don't know what test/example failed for me to add this, or if I just made an assumption, but I'm going to need to tag @hiker as I'm not sure I know what the result of this function should be for TYPE_INFO elements.

On the create_type_map test I added `1_wp` to the list of expressions, and my assumption is we don't need the `wp` symbol in the type_map? Which is what the above code is attempting (and failing to do). Is that assumption correct, or should we still have `wp` in the type map?